docs/processing-performance.asciidoc

```diff
@@ -5,40 +5,110 @@ APM Server performance depends on a number of factors: memory and CPU available,
 network latency, transaction sizes, workload patterns,
 agent and server settings, versions, and protocol.
 
-Let's look at a simple example that makes the following assumptions:
+// Assumptions
+// TBD
 
-* The load is generated in the same region as where APM Server and {es} are deployed.
-* We're using the default settings in cloud.
-* A small number of agents are reporting.
+// Method
+To help you understand how sizing APM Server impacts performance, we tested several scenarios:
 
-This leaves us with relevant variables like payload and instance sizes.
-See the table below for approximations.
-As a reminder, events are
+* Using the default hardware template on AWS, GCP and Azure on {ecloud}.
+* Comparing the performance when using OpenTelemetry (OTel) events on at least one of the service providers listed above (in this case, GCP).
+* For each hardware template, testing with several sizes: 1 GB, 4 GB, 8 GB, and 32 GB.
+* For each size, using a fixed number of APM agents: 10 agents for 1 GB, 30 agents for 4 GB, 60 agents for 8 GB, and 240 agents for 32 GB.
+* In all scenarios, using medium sized events. Events include
 <<data-model-transactions,transactions>> and
 <<data-model-spans,spans>>.
 
+// This leaves us with relevant variables like payload and instance sizes.
+
+// Results
+You can use the results of our tests to guide your sizing decisions, however, performance will vary based on factors unique to your use case like your specific setup, the size of APM event data, and the exact number of agents.
+
+:hardbreaks-option:
+
 [options="header"]
-|=======================================================================
-|Transaction/Instance |512 MB Instance |2 GB Instance |8 GB Instance
-|Small transactions
+|====
+| Profile / Cloud | AWS | Azure | GCP (Agent) | GCP (OTel)
```

Comment:
Can we exclude these `GCP (OTEL)` column? It's misleading and customers will be confused.These are details on how we run the benchmarks. Our OTel benchmarks use a barebones OTel transaction which isn't transferrable for customers.

Reply:
yes, please keep otel out for now